src/daily-auths-report.tsx

```diff
@@ -190,10 +222,13 @@ function DailyAuthsReport(): VNode {
   return (
     <div>
       <div className="chart-wrapper" ref={ref} />
-      <Table data={tabulate(data || [], agency, ial)} numberFormatter={format(",")} />
+      <Table
+        data={agency ? tabulate(data || [], agency, ial) : tabulateSumByAgency(data || [], ial)}
```

Comment:
Maybe `results` should be an optional argument?

```suggestion
        data={agency ? tabulate(data, agency, ial) : tabulateSumByAgency(data, ial)}
```

Reply:
Good idea, updated in 81de3fa60db6c6c20c85a9d5303c4b991c838a55